packages/server/lib/automation/cookies.ts

```diff
@@ -130,7 +134,7 @@ export class Cookies {
     })
   }
 
-  setCookies (cookies, automate) {
+  setCookies (cookies, automate, eventName = 'set:cookies') {
```

Comment:
Not sure if these typings are correct, but I think they go a long way to help understand how the function works.
```suggestion
  setCookies (cookies: AutomationCookie[], automate: (eventName: string, cookies: AutomationCookie[]) => Bluebird.Promise<AutomationCookie[]>, eventName: 'set:cookies' | 'add:cookies' = 'set:cookies') {
```

Reply:
Addressed in [`db8684a` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/db8684ab6c3f49fda2961344dcb23972701465b9)